app/digidoo/hospitals/create/template.hbs

```diff
@@ -0,0 +1,55 @@
+<h2>Create</h2>
+<form action="/hospitalsmvc/create" method="post">
```

Comment:
This form still doesn't have an action handler so the browser is going to continue to do it's standard action and it will reload the page while making a post request. Instead, you will want to intercept the submit action and have the model try to save.

Look at this form from 6.4-but-ember as an example: https://github.com/TIY-LR-FEE-2015-June/6.4-but-ember/blob/master/app/templates/components/cms-form.hbs#L1